README.md

```diff
@@ -42,28 +42,20 @@ If things aren't working, double and triple check that the user plugin is disabl
 
 9. (Optional) - Copy the `kolibri_instant_schools_plugin/assets/src/views/about/views/about` to `$(KOLIBRI_HOME)/content/databases`. Note that the about folders are indeed nested and that is not a typo. 
 
-How do I test SMPP locally?
-----------------------------
+Env Vars
+--------
 
-1. From the root of the `kolibri_instant_schools_plugin` project, `cd kolibri_instant_schools_plugin/smpp/SMPPSim` 
-2. Now, unzip the file: `unzip SMPPSim.zip` which should create a new folder `SMPPSim` 
-3. Now, `cd SMPPSim` and you can run `sudo sh startsmppsim.sh` from that folder anytime to test SMPP.
+The following environment variables are *required*:
```

Comment:
Noting as I'm going through old PRs to decide whether to merge or not... that this is not relevant during development although the server will not start in prod w/out the vars here.